bpf/bpf_lxc.c

```diff
@@ -672,9 +686,13 @@ static inline int handle_ipv4_from_lxc(struct __sk_buff *skb, __u32 *dstID)
 
 __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV4_FROM_LXC) int tail_handle_ipv4(struct __sk_buff *skb)
 {
+    struct ep_config *cfg = lookup_ep_config();
```

Comment:
Sigh, I think my spacing is off. @joestringer what is the appropriate spacing for the C code?

Reply:
@ianvernon It's tabs indented 8 characters, just like our Go code.

Reply:
I think I caught all of the irregular whitespace.